README.md

```diff
@@ -34,17 +32,33 @@ git clone https://github.com/<username>/practical-programming-in-chemistry-miles
 
 Replace `<username>` with your GitHub username.
```

Comment:
Replace `username` instead of <username>?

Reply:
updated this in [commit](https://github.com/schwallergroup/practical-programming-in-chemistry-milestones/pull/3/commits/c55a186730c7c77fbbaa5083517427d228e23787)